study-builder/studies/osteo/study.conf

```diff
@@ -2544,6 +2542,110 @@
 
     ## events at age-up
 
+    # reconsent events
```

Comment:
Sorry for nitpicking... can you move this label back where it was (after L2425)? This whole section is for `reconsent events` whereas this subsection is for `events at age-up`.